docs/src/navigation/navigationTabs/navigationTabsSample.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { NavigationTabs } from 'react-cm-ui';
+
+const items = [
```

Comment:
The ITEMS ARRAY seems to be a long array, would it be good to include the whole mocked up array in the docs snippet? or should we just include it in the first sample and then referencing it for the rest of the samples?

Reply:
It's not _so_ bad; I think it is okay to include it.  Knowing how to structure the `items` array is important to knowing how to use the component ... right?  That's something we want to emphasize in the code samples in the documentation.

Reply:
Docs updated, I added a simpler `items `mockup.

Reply:
Lookin' good to me.